README.md

```diff
@@ -12,17 +12,29 @@ This is built on [React.js](https://reactjs.org/) and is deployed on Vercel
 
 1. Join the [#project_codesydneywebsite Slack channel](https://codesydney.slack.com/archives/C030XRCQ5A5) and start contributing
 2. Join the [#project_codesydneywebsite Discord channel](https://discordapp.com/channels/1041795715757789204/1043385733085413396)
-3. We have open issues that can be worked on in the [Issues tab](https://github.com/codesydney/codesydneywebsite/issues). You can see how these issues are progressing on our [Kanban board](https://github.com/codesydney/codesydneywebsite/projects/1).
-4. Visit our [Code.Sydney Playbook](https://github.com/codesydney/code-sydney-playbook/blob/main/docs/pull-request.md) here for a full guide
+3. We have open issues that can be worked on in the [Issues tab](https://github.com/codesydney/codesydneywebsite/issues).
+4. You can see how these issues are progressing on our [Kanban board](https://github.com/codesydney/codesydneywebsite/projects/1).
+5. Visit our [Code.Sydney Playbook](https://code-sydney-playbook.vercel.app/docs/pull-request) here for a full guide
 
 ### Setting up your machine for local development
 
-1. If you need any help to fork this repository (https://github.com/firstcontributions/first-contributions)
-2. Clone this to your local machine `git clone git@github.com:codesydney/codesydneywebsite.git` you can copy your own code url after fork
-3. `npm install`
-4. If receiving any error install NVM by following (https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/)
-5. Make sure you are in codesydneywebsite project directory type in folowing command:-
-6. `nvm install 14` > `nvm use 14` > `npm i` > `npm start`
-7. Access the site using your browser of choice via http://localhost:3000/
+### Forking and Creating Your Branch
 
-_Note: You may need to downgrade to an older version of Node.js such as https://nodejs.org/download/release/latest-v14.x/ to get everything running._
+1. How to fork this repository (https://github.com/firstcontributions/first-contributions)
+2. Copy code url from your forked version of project
+3. Avoid cloning directly from codesydney repo
+4. CD into directory on cmd terminal where you would like to store project files
+5. Finally enter following command to clone repo `git clone [url]`
+
+### Installing the Project
+
+1. If you are not using Node.js v14 install NVM by following (https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/)
+2. Make sure you are in codesydneywebsite project directory on your cmd terminal then type in folowing command:-
+3. `nvm install 14` > `nvm use 14` > `npm i`
+
+### Running the Project
+
```

Comment:
Can you please add a new line here to 

1. Change to Node v14 using NVM with `nvm use 14`.

When the use runs the project without changing node versions, the project won't run.